zeus/run/dataloader.py

```diff
@@ -276,6 +349,19 @@ def __init__(
         if not self._is_train:
             ZeusDataLoader.eval_num_samples = self.num_samples
 
+        # If the number of iterations in one epoch (`num_samples`) is smaller than or equal
+        # to one profile window (`warmup_iters + profile_iters`), we will not be able to
+        # profile for any power limit. So, we scale the profile window to fit in one epoch.
+        # We also avoid using the last batch of one epoch, becasue when `drop_last == True`,
+        # the last batch will be smaller. This usually happens with large batch size on
+        # small datasets, eg. CIFAR100.
+        if self.warmup_iter + self.profile_iter >= self.num_samples:
```

Comment:
Could you print out a message that this case was triggered and the new warmup/profile iters?